src/GitHub.VisualStudio/UI/Views/Controls/RepositoryPublishControl.xaml.cs

```diff
@@ -30,7 +30,13 @@ public RepositoryPublishControl()
                 d(this.Bind(ViewModel, vm => vm.RepositoryName, v => v.nameText.Text));
                 
                 d(this.Bind(ViewModel, vm => vm.Description, v => v.description.Text));
-                d(this.Bind(ViewModel, vm => vm.KeepPrivate, v => v.makePrivate.IsChecked));
+
+                d(Observable.FromEventPattern<RoutedEventArgs>(makePrivate, "Checked")
+                    .Subscribe(_ => ViewModel.KeepPrivate = true));
+
+                d(Observable.FromEventPattern<RoutedEventArgs>(makePrivate, "Unchecked")
+                    .Subscribe(_ => ViewModel.KeepPrivate = false));
+
```

Comment:
Let's add a comment on why this abomination is required as opposed to the mores straightforward code that works elsewhere. :smile: